src/workflow_components/composite_action.py

```diff
@@ -27,6 +28,42 @@ def get_or_create_composite_action(path: str) -> "CompositeAction":
     return obj
 
 
+class CompositeActionInput(GraphObject):
+    __primarykey__ = "_id"
+
+    _id = Property()
+    name = Property()
+    default = Property()
+    description = Property()
+    required = Property()
```

Comment:
Add path property; It is the convention and it will be crucial for integration testing by path.